clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -379,6 +379,30 @@ def PtrStrideOp : CIR_Op<"ptr_stride",
   let hasVerifier = 0;
 }
 
+//===----------------------------------------------------------------------===//
+// PoisonOp
+//===----------------------------------------------------------------------===//
+def PoisonOp : CIR_Op<"poison", [Pure]> {
+  let summary = "Creates a poison value of CIR type";
+
+  let description = [{
+    Unlike LLVM IR, MLIR does not have first-class poison values. Such values
```

Comment:
I'd rewrite the description to: `Creates a poison value of the specified CIR type. This is currently used to forward potential erroneous behavior down to LLVM when lowering CIR`.

Reply:
Sure, I'll update the description if we decide to introduce this Op.